R/load_db.R

```diff
@@ -30,8 +30,8 @@ load_db <- function(db_connection, csv_path_dir, cdm_metadata,
     # Get metadata for the current table
     table_metadata <- cdm_metadata[TABLE %in% table_name_pattern]
     # Identify date columns that are mandatory and date
-    date_cols <- table_metadata[str_detect(Format, "yyyymmdd") == TRUE &
-                                  str_detect(Mandatory, "Yes") == TRUE, Variable]
+    date_cols <- table_metadata[stringr::str_detect(Format, "yyyymmdd") == TRUE &
```

Comment:
Good catch

Reply:
I caught it only because the tests failed ;)